Content.Server/Bed/Components/StasisBedComponent.cs

```diff
@@ -1,13 +1,21 @@
+using Content.Shared.Construction.Prototypes;
+using Robust.Shared.Serialization.TypeSerializers.Implementations.Custom.Prototype;
+
 namespace Content.Server.Bed.Components
 {
     [RegisterComponent]
     public sealed class StasisBedComponent : Component
     {
+        [DataField("baseMultiplier", required: true), ViewVariables(VVAccess.ReadWrite)]
+        public float BaseMultiplier = 10f;
+
         /// <summary>
         /// What the metabolic update rate will be multiplied by (higher = slower metabolism)
         /// </summary>
-        [DataField("multiplier", required: true)]
         [ViewVariables(VVAccess.ReadWrite)]
         public float Multiplier = 10f;
```

Comment:
shouldnt you be setting this on init if its being split

Reply:
Pretty sure this is not needed, as the `RefreshPartsEvent` is raised on the machine on MapInit when its internal containers are filled with the default part entities.

Reply:
ok yeah